components/ClientNavMenu.tsx

```
@@ -0,0 +1,72 @@
+                  ? "bg-yellow-400 text-black font-medium hover:bg-yellow-500"
+                  : "bg-white/10 text-foreground hover:bg-white/20"
+                  }`}
+              >
+                <Link href={item.href} className="flex items-center gap-2">
+                  {item.text}
+                  <Square size={16} fill="currentColor" />
+                </Link>
+              </Button>
+            );
+          })}
+        </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)